packages/@react-spectrum/toast/docs/Toast.mdx

```diff
@@ -58,7 +58,7 @@ Then, queue a toast from anywhere:
 
 Toasts are triggered using one of the methods of <TypeLink links={docs.links} type={docs.exports.ToastQueue} />. A &lt;<TypeLink links={docs.links} type={docs.exports.ToastContainer} />&gt; element must be rendered at the root of your app in order to display the queued toasts.
 
-Toasts are shown according to a priority queue, depending on their variant. Actionable toasts are prioritized over non-actionable toasts, and errors are prioritized over other types of notifications. Only one toast is displayed at a time. See the [Spectrum design docs](https://spectrum.adobe.com/page/toast/#Priority-queue) for full information on toast priorities.
+Toasts are shown according to the order they are added from the bottommost position. Since all opens Toasts are displayed until dismissed, please be mindful of how often toasts are triggered, see [Spectrum design docs](https://spectrum.corp.adobe.com/page/toast/#Too-many-toasts).
```

Comment:
```suggestion
Toasts are shown according to the order they are added, with the most recent toast appearing at the bottom of the stack. Since all some Toasts may be displayed until dismissed, please be mindful of how often toasts are triggered, see [Spectrum design docs](https://spectrum.adobe.com/page/toast/#Too-many-toasts).
```
Some small changes for clarity with auto dismissing toasts in mind